tests/common/utils/test_prompt.py

```diff
@@ -46,6 +42,13 @@ def test_prompt_terraform_aws_resource(capsys: CaptureFixture[str]):
     for exp in expected:
         assert exp in captured.out
 
+    # Remove unit test files
```

Comment:
Previously, the files generated in these tests were not removed and they were in a weird directory. This meant that every time I ran pytest, i couldn't run `git add .` afterwards without adding on the files that were generated as part of this test.

These adjustments remove that issue. The other unit tests follow this format.